alerts/kube_apiserver.libsonnet

```diff
@@ -145,6 +145,32 @@ local utils = import 'utils.libsonnet';
               message: 'A client certificate used to authenticate to the apiserver is expiring in less than %s.' % (utils.humanizeSeconds($._config.certExpirationCriticalSeconds)),
             },
           },
+          {
+            alert: 'AggregatedAPIErrors',
+            expr: |||
+              sum by(name, namespace)(increase(aggregator_unavailable_apiservice_count[5m])) > 2
```

Comment:
Seems like aggregator_unavailable_apiservice_count is a counter, so the name suffix should be _total instead, we should open an issue in the upstream repo for this to replace the name with aggregator_unavailable_apiservice_total.

Reply:
oh, right, I think that's the convention, sure np I can open an upstream PR.

Reply:
fixed in https://github.com/kubernetes/kubernetes/pull/88156

Reply:
wow, nice, thanks!